app/components/CV/CV.tsx

```diff
@@ -1,14 +1,18 @@
+import { CVData } from '~/lib/schemas/classicCVTemplateSchema'
 import { Experience } from './experience.js'
 import { LeftSideSections } from './leftSideSections.js'
 import { Section } from './section.js'
-import { CvDataType } from './types.js'
 
-const CV = (data: CvDataType) => {
+type CVDisplayProps = {
+  data: CVData
+}
+
+export const CV: React.FC<CVDisplayProps> = ({ data }) => {
   return (
     <div className="flex w-2/3 pt-12 border-stone-700 border-2 pb-16">
       {/* left side */}
       <div className="h-full w-2/5 flex flex-col px-8 justify-around">
```

Comment:
**praise**: Dynamic rendering of the name from data enhances flexibility and personalization.